gallery/codeviewer_cli/README.md

```diff
@@ -5,10 +5,14 @@ A command-line application to highlight dart source code.
 Code segments are highlighted before the app is compiled.
 This is done because the highlighting process can take 300ms to finish, creating a noticeable delay when the demo switches to code page.
 
-The highlighter takes all files in the `gallery/lib/demos/` folder and scans each.
-Highlighted code widgets are stored in the `gallery/lib/codeviewer/code_segments.dart` file.
-Under the root directory, run `make update-code-segments` to run the highlighter.
+The highlighter takes all files in the `gallery/lib/demos/` folder and scans each. Highlighted code widgets are stored in the `gallery/lib/codeviewer/code_segments.dart` file.
```

Comment:
nit: While there are exceptions, markdown in the repo normally uses an ~80 character line break.

Reply:
TIL, makes sense

Makes me think we should have something like https://github.com/markdownlint/markdownlint done in Dart, but for now I'll update it here manually.